pkg/app/master/commands/clifvgetter.go

```diff
@@ -407,3 +408,30 @@ func GetDockerClientConfig(ctx *cli.Context) *config.DockerClient {
 
 	return config
 }
+
+func validateAndCleanEnvVariables(envList []string, xc *app.ExecutionContext) []string {
+	xc.Out.Info("Validating and cleaning environment variables")
+
+	envStaging := envList[:0]
+	for i, env := range envList {
+		envKeyValue := strings.Split(env, "=")
+		key := envKeyValue[0]
+		value := envKeyValue[1]
```

Comment:
this will cause a crash if the `env` is not well formatted and it doesn't have `=` in it.